### ISSUE TITLE
createError.js when attempting to POST to Api

### ISSUE THREAD

**Issue by sneadthaman** [susanBuck/e28-fall20#63]
<img width="1392" alt="Screen Shot 2020-12-01 at 12 52 30 PM" src="https://user-images.githubusercontent.com/30006142/100777682-20b8d700-33d4-11eb-97d0-a8438c809a38.png">

All I'm attempting to do at the moment is post the form data to my new API endpoint '/holidays' and console.log(response.data)

Here is my method:
`addUserDay() {
        axios.post('/holidays', this.userHoliday).then((response) => {
          if (response.data.errors) {
          this.errors = response.data.errors;
          } else {
          //this.$emit('update-holidays');
          console.log(response.data)
          }
        }
        )}
    }`

**Comment by susanBuck** [comment]
Hi @sneadthaman -

Can you share the Github URL to this code?

**Comment by sneadthaman** [comment]
https://github.com/sneadthaman/e28_new/blob/master/p2/forgetmenot/src/components/UserDates.vue

**Comment by susanBuck** [comment]
Ok - I cloned locally to test.

I noticed when I click the button to _Add a day_, it was causing the page to refresh. This is because the form inputs are wrapped in a `<form></form>` element, so the default browser behavior when the button is clicked is to attempt to submit the form to the server.

To override this, add the `.prevent` modifier to your button:

```html
<button @click.prevent="addUserDay">Add a Day</button>
```

(Ref: https://vuejs.org/v2/guide/events.html#Event-Modifiers)

Once I did this, I was able to successfully see the response logged to the console ([screenshot for reference](https://s3.amazonaws.com/screenshots001/2020-12-01-16-22-37.png)).
